charts/gsp-cluster/templates/03-namespaces/namespace.yaml

```diff
@@ -379,12 +379,10 @@ spec:
                 exit 1
               fi
               echo "configuring kubectl..."
-              echo "${KUBERNETES_SERVICE_ACCOUNT}" | jq -r .["ca.crt"] > ca.crt
-              kubectl config set-cluster self --server=https://kubernetes.default --certificate-authority=ca.crt
-              kubectl config set-credentials deployer --token "${KUBERNETES_TOKEN}"
-              kubectl config set-context deployer --user deployer --cluster self
+
+              setup-kube-deployer
               kubectl config set-context deployer --namespace "${RELEASE_NAMESPACE}"
```

Comment:
i don't know if it's a problem to switch the order of `set-context` and `use-context` here.

Reply:
I expect it'll be fine. We'll find out when we test.